docs/examples/full_chain_odd.md

```diff
@@ -1 +1,123 @@
-# ODD full chain
+# Walkthrough of the OpenDataDetector full chain example
+
+The OpenDataDatector (ODD) is fictitious silicon detector which resides in its own repository on [GitLab](https://gitlab.cern.ch/acts/OpenDataDetector). It is used for testing and as a performance baseline in Acts.
+
+Our full chain ODD example is written in Python and can be found [here](https://github.com/acts-project/acts/blob/main/Examples/Scripts/Python/full_chain_odd.py).
+
+The first step is to load the ODD detector description and to construct the detector. `getOpenDataDetectorDirectory` gives us the ODD folder within the `thirdparty` directory in Acts. We load our prefered material map and provide it to the detector construction `getOpenDataDetector`.
+
+```python
+oddDir = getOpenDataDetectorDirectory()
+
+oddMaterialMap = oddDir / "data/odd-material-maps.root"
+oddMaterialDeco = acts.IMaterialDecorator.fromFile(oddMaterialMap)
+
+detector, trackingGeometry, decorators = getOpenDataDetector(
+    getOpenDataDetectorDirectory(),
+    mdecorator=oddMaterialDeco,
+)
+```
+
+In our simple example we assume a homogenous magnetic field along the beam axis with 2 T. The magnetic field is passed to all the different algorithms in our simulation and the reconstruction pipeline.
+
+```python
+field = acts.ConstantBField(acts.Vector3(0.0, 0.0, 2.0 * u.T))
+```
+
+The simulation step involves random processes and in order to get the same results across different executions we provide our prefered random number generator with an arbitrary seed.
+
+```python
+rnd = acts.examples.RandomNumbers(seed=42)
+```
+
+All simulation and reconstruction pipelines in Acts begin with a `Sequencer`. It controls the execution of the different algorithms in the chain. We provide the number of events, the number of threads to use (`-1` to use all the machine's cores) and the desired log level.
+
+```python
+s = acts.examples.Sequencer(events=100, numThreads=-1, logLevel=acts.logging.INFO)
+```
+
+Our first simulation step is the particle gun. It spawns particles and their initial parameters, like position and momentum, inside our detecor.
+
+In our simple example we generate a single muon with random charge (e.g. muon or anti-muon) with 1-10 GeV with uniform pseudorapidity from -3 to 3.
+
+Relativistic muons are hardly deflected in the detector and will keep most of their energy which makes them almost ideal for track reconstruction.
+
+```python
+addParticleGun(
+    s,
+    MomentumConfig(1.0 * u.GeV, 10.0 * u.GeV, True),
+    EtaConfig(-3.0, 3.0, True),
+    ParticleConfig(1, acts.PdgParticle.eMuon, True),
```

Comment:
```suggestion
    MomentumConfig(1.0 * u.GeV, 10.0 * u.GeV, transverse=True),
    EtaConfig(-3.0, 3.0, uniform=True),
    ParticleConfig(1, acts.PdgParticle.eMuon, randomizeCharge=True),
```
Sorry, this isn't just for this doc, but I noticed it again here. We could also add labels to the other parameters, but they are probably obvious and this gives a good balance between clarity and brevity.
(I didn't check this syntax, but I hope that's right.)

Reply:
oh yeah that makes sense thank you! I will also add it to https://github.com/acts-project/acts/pull/1404